packages/core/src/html/NodeProcessor.js

```diff
@@ -315,6 +326,68 @@ class NodeProcessor {
     return node;
   }
 
+  /*
+   * Syntax is (space)#char#(any text)##(space)
+   * If we have #char# without ## or ## without #char# we don't replace
+   * To escape start do \#char#, so it will show up as #char# (escaping also works with any front char)
+   * To escape end do \##, which will show up as ##
+   */
+  preprocessMarkdown(content) {
```

Comment:
@weiquu Sorry I didn't manage to take a look at the implementation earlier. I assumed this was implemented similarly to one of our current double-delimiter plugins.

I don't think this approach is quite right, writing a markdown parser / processor isn't trivial. We should be tapping directly into `markdownIt`'s parsing / [plugin](https://github.com/markdown-it/markdown-it/tree/master/docs) system.

There are far too many edge cases from trying to parse / handle the file from scratch (other than just the few pointed out above...)

e.g. comments, for one

```
/*
#r#...##
*/
```

Reply:
Tldr basically the same as @tlylt's [comment ](https://github.com/MarkBind/markbind/pull/1988#issuecomment-1209207496)above, I haven't read that when I posted this 😶